tests/postprocessing_test.py

```diff
@@ -318,6 +319,14 @@ def test_steep_upslope(self):
             steep_upslope[-1]
             )
 
+    def test_consecutive_manoeuvres(self):
+        valid_manoeuvres, double_dips = consecutive_manoeuvres(
+            y_t_paw,1,9,fs_vent,pocc_peaks_valid)
```

Comment:
Consider entering variables as:
signal=y_t_paw,start_s=1,end_s=9,fs=fs_vent,peaks_s=pocc_peaks_valid
If the test fails, it's easier to see what inputs are used and why it fails